### PR TITLE
fix: adjust message composer to design specs

### PR DESCRIPTION
**Pull request by trOnk12**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [ ] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`
  - [ ] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

- adjust the message composer according to design specs 

### Attachments (Optional)

image

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [1](https://10.10.124.134/job/wire-android-reloaded/job/PR-521/1/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3141736)(3141736)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3141736)(3141736)-dev-release.apk)

File: app/src/main/kotlin/com/wire/android/ui/home/conversations/MessageItem.kt [hunk 1]
```diff
@@ -64,13 +63,15 @@ fun MessageItem(
                     onLongClick = onLongClicked
                 )
         ) {
+            Spacer(Modifier.padding(start = 2.dp))
             UserProfileAvatar(
                 userAvatarAsset = message.user.avatarAsset,
                 status = message.user.availabilityStatus
             )
+            Spacer(Modifier.padding(start = 12.dp))
```
Comment (yamilmedina) [inline app/src/main/kotlin/com/wire/android/ui/home/conversations/MessageItem.kt:71]:
```suggestion
            Spacer(Modifier.padding(start = dimensions().spacing12x))
```

File: app/src/main/kotlin/com/wire/android/ui/home/conversations/MessageItem.kt [hunk 2]
```diff
@@ -64,13 +63,15 @@ fun MessageItem(
                     onLongClick = onLongClicked
                 )
         ) {
+            Spacer(Modifier.padding(start = 2.dp))
```
Comment (yamilmedina) [inline app/src/main/kotlin/com/wire/android/ui/home/conversations/MessageItem.kt:66]:
```suggestion
            Spacer(Modifier.padding(start = dimensions().spacing2x))
```

**Review by yamilmedina — COMMENTED** [review]
Looks good to me, just a couple of comments that we can take the advantage to polish it 🎸 🎶

**Comment by AndroidBob** [comment]
Build [2](https://10.10.124.134/job/wire-android-reloaded/job/PR-521/2/) **failed**.

File: app/src/main/res/drawable/ic_take_picture.xml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```
[the remark this answers is not in the document]
Comment (trOnk12) [reply app/src/main/res/drawable/ic_take_picture.xml:1]: I actually did not know what this Icon means and I assumed it is only for taking a picture :P I am going to ask Wolfgang on what does icon does and rename it

[the remark this answers is not in the document]
Comment (trOnk12) [reply app/src/main/res/drawable/ic_take_picture.xml:1]: ok I actually checked what it is an the icon is for pinging the users rename it :P @yamilmedina

**Comment by AndroidBob** [comment]
Build [3](https://10.10.124.134/job/wire-android-reloaded/job/PR-521/3/) **failed**.

**Comment by AndroidBob** [comment]
Build [4](https://10.10.124.134/job/wire-android-reloaded/job/PR-521/4/) **failed**.

**Comment by AndroidBob** [comment]
Build [5](https://10.10.124.134/job/wire-android-reloaded/job/PR-521/5/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3148047)(3148047)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3148047)(3148047)-dev-release.apk)

**Pull request merged**